src/PProf.jl

```diff
@@ -216,7 +217,9 @@ function pprof(data::Union{Nothing, Vector{UInt}} = nothing,
                 file = string(meth.file)
                 io = IOBuffer()
                 Base.show_tuple_as_call(io, meth.name, linfo.specTypes)
-                full_name_with_args = _escape_name_for_pprof(String(take!(io)))
+                call_str = String(take!(io))
+                # add module name as well
+                full_name_with_args = _escape_name_for_pprof("$(meth.module).$call_str")
```

Comment:
adding the module name here should be the only user-visible change in this PR — it's pretty essential to get frames to show up right in DD since it essentially does

```
package, func = split(name, "."; limit=2)
```

e.g. for this one:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/7341/198555659-8a408e4a-c035-41b7-90c5-79035e980899.png">

Without the module name it would parse as
* package: `DatadogProfileUploader.profile_and_upload(DatadogProfileUploader`
* func: `DDConfig, typeof(myfunc))`

If people don't want the behavior to change though, I could put this behind an option.